### PR TITLE
Currencies for 2017, fix for new weird NACP format of declarations, script to export all uuids to file to reimport everything to ES

### PR DESCRIPTION
**Pull request by dchaplinsky**

### CODE REVIEW

File: utils/get_all_uuids.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    args = parser.parse_args()
+
+    db_config = {
+        'user': args.username,
+        'password': args.password,
+        'url': args.endpoint,
+        'name': args.dbname
+    }
+
+    # Ensure our DB exists before actually importing
```
Comment (excieve) [inline utils/get_all_uuids.py]: I don't think you need this. What's the point of exporting anything if DB doesn't exist yet?

File: utils/requirements.txt [hunk 2]
```diff
@@ -4,3 +4,4 @@ elasticsearch==5.4.0
 glob2==0.5
 dpath==1.4.0
```
Comment (excieve) [inline utils/requirements.txt]: I guess this can be removed now?

**Comment by dchaplinsky** [comment]
Check it out now

**Comment by excieve** [comment]
Thanks!

**Pull request merged**